### PR TITLE
Update /what-is-kubernetes as per copydoc

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

*Waiting on feedback from pdm*
- Update /what-is-kubernetes as per [copydoc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-11353.demos.haus/kubernetes/what-is-kubernetes
    - Be sure to test on mobile, tablet and desktop screen sizes
-  Compare against [copydoc](https://docs.google.com/document/d/19ZAsyujRHjOBIKP15F-4Z9L24KpfoYrx0YpL-0gO6sE/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4904

### CODE REVIEW

**Comment by davegoddard42** [comment]
LGTM @petesfrench - nice work.

File: templates/kubernetes/what-is-kubernetes.html [hunk 1]
```diff
@@ -710,92 +850,6 @@ <h4 class="p-card__title">
...
-              }}
-              <h3 class="p-heading-icon__title p-heading--4">Whitepaper</h3>
-            </div>
-          </div>
-          <p><a href="/engage/kubernetes-deployment-enterprise-whitepaper" class="p-link--inverted">Five strategies to accelerate Kubernetes deployment in the enterprise</a></p>
-      </div>
-    </div>
-  </section>
-
   <section class="p-strip">
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:853]:
```suggestion
  <section class="p-strip--light">
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 2]
```diff
@@ -20,13 +20,10 @@ <h1>
...
         </p>
         <p>
-          Interested in Canonical Kubernetes?
-        </p>
-        <p>
-          <a class="p-button--positive js-invoke-modal" href="/kubernetes/contact-us">Get in touch</a>
+          Looking to build your Kubernetes strategy? <a href="/engage/kubernetes-deployment-enterprise-whitepaper">Read our buyer's guide&nbsp;&rsaquo;</a>
         </p>
       </div>
       <div class="col-5 u-align--center u-vertically-center">
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:29]:
```suggestion
      <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 3]
```diff
@@ -46,28 +43,73 @@ <h1>
...
+    </p>
+  </div>
   <div class="row u-sv3">
-    <div class="col-8 u-sv3">
-      <h2>What is container orchestration?</h2>
-      <p>
-        Container orchestration is about managing the lifecycle of containers, particularly in large, dynamic environments. It automates the deployment, networking, scaling, and availability of containerised workloads and services. Running containers - which are lightweight and usually ephemeral by nature - in small numbers, is easy enough to be done manually. However, managing them at scale in production environments can be a significant challenge without the automation that container orchestration platforms offer. Kubernetes has become the standard for container orchestration in the enterprise world.
-      </p>
+    <div class="col-4">
+      <div class="u-sv3">
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:63]:
```suggestion
      <div class="u-sv3 u-hide--small">
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 4]
```diff
@@ -46,28 +43,73 @@ <h1>
...
-  <div class="row">
-    <div class="col-8">
-      <div class="p-notification--information">
-        <div class="p-notification__content">
-          <p class="p-notification__message">
-            We asked Devs, DevOps and businesses to tell us how they are using Kubernetes, and it made for a fascinating read. <a href="http://juju.is/cloud-native-kubernetes-usage-report-2021" class="p-link--external">Read our findings</a>
-          </p>
-        </div>
+    <div class="col-4">
+      <div class="u-sv3">
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:77]:
```suggestion
      <div class="u-sv3 u-hide--small">
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 5]
```diff
@@ -46,28 +43,73 @@ <h1>
...
+          height="100",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
       </div>
+      <p class="p-heading--4">46% Modernizing infrastructure</p>
+    </div>
+    <div class="col-4">
+      <div class="u-sv3">
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:91]:
```suggestion
      <div class="u-sv3 u-hide--small">
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 6]
```diff
@@ -143,12 +185,96 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
...
+              width="32",
+              height="28",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+              ) | safe
+            }}
+            <h3 class="p-heading--4">Webinar on demand</h3>
+          </div>
+          <p><a class="p-link--inverted" href="/engage/kubernetes-use-cases-webinar">Enterprise Kubernetes use cases: 4 real-world stories</a></h3>
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:227]:
```suggestion
          <p><a class="p-link--inverted" href="/engage/kubernetes-use-cases-webinar">Enterprise Kubernetes use cases: 4 real-world stories</a></p>
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 7]
```diff
@@ -143,12 +185,96 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
...
+              width="32",
+              height="28",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+              ) | safe
+            }}
+            <h3 class="p-heading--4">Video</h3>
+          </div>
+          <p><a class="p-link--inverted" href="https://www.youtube.com/watch?v=tG5kqnnytIg">Canonical Kubernetes powered by Ubuntu: Charmed Kubernetes & MicroK8s</a></h3>
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:246]:
```suggestion
          <p><a class="p-link--inverted" href="https://www.youtube.com/watch?v=tG5kqnnytIg">Canonical Kubernetes powered by Ubuntu: Charmed Kubernetes & MicroK8s</a></p>
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 8]
```diff
@@ -143,12 +185,96 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
...
+              width="32",
+              height="28",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+              ) | safe
+            }}
+            <h3 class="p-heading--4">Whitepaper</h3>
+          </div>
+          <p><a class="p-link--inverted" href="https://ubuntu.com/engage/kubernetes-deployment-enterprise-whitepaper">Five strategies to accelerate Kubernetes deployment in the enterprise</a></h3>
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:265]:
```suggestion
          <p><a class="p-link--inverted" href="https://ubuntu.com/engage/kubernetes-deployment-enterprise-whitepaper">Five strategies to accelerate Kubernetes deployment in the enterprise</a></p>
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 9]
```diff
@@ -143,12 +185,96 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
...
+              width="32",
+              height="28",
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+              ) | safe
+            }}
+            <h3 class="p-heading--4">Datasheet</h3>
+          </div>
+          <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></h3>
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:208]:
```suggestion
          <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></p>
```

File: templates/kubernetes/what-is-kubernetes.html [hunk 10]
```diff
@@ -143,12 +185,96 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
   </div>
 </section>
 
-<section class="p-strip--light">
+<section class="p-strip--suru">
+  <div class="u-fixed-width">
+    <h2>Kubernetes resources</h2>
+    <div class="row p-divider u-no-padding--left u-no-padding--right is-dark u-equal-height">
+      <div class="col-3 p-divider__block">
+        <div class="p-heading-icon--muted">
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:193]: I think `p-heading-icon--muted` class has been deprecated. Could we update the [heading icon](https://vanillaframework.io/docs/patterns/heading-icon) component?

File: templates/kubernetes/what-is-kubernetes.html [hunk 11]
```diff
@@ -680,7 +820,7 @@ <h4 class="p-card__title">
             Charmed Kubernetes has been tested across the widest range of infrastructures, and deploys on bare-metal, private and public clouds. Along with our Kubernetes, Canonical supports a rich ecosystem of different pieces of software that can be integrated from the stack up. Canonical is one of Microsoft's primary development partners as Ubuntu is the operating system of choice on AKS.
           </p>
           <p>
-            <a href="/kubernetes/docs/quickstart">
+            <a href="https://ubuntu.com/kubernetes/install">
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:823]:
```suggestion
            <a href="/kubernetes/install">
```

**Review by minkyngkm — CHANGES_REQUESTED** [review]
Mostly looking good @petesfrench! I just made some comments about `u-hide--small` class and the heading icon component. :) 

Also, it would look better if the `p-link--external` can be removed in the `shared/contextual_footers/_contextual_footer.html` file!

File: templates/kubernetes/what-is-kubernetes.html [hunk 12]
```diff
@@ -143,12 +185,88 @@ <h2 class="u-sv3">Kubernetes advantages</h2>
...
+            width="32",
+            height="28",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
+            ) | safe
+          }}
+          <h3 class="p-heading--4">Datasheet</h3>
+        </div>
+        <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></p>
```
Comment (minkyngkm) [inline templates/kubernetes/what-is-kubernetes.html:207]: It looks good now but would it be better if it's wrapped by `p-heading-icon--small` or `p-heading-icon`. Feel free to update it if that's better you think! 

```suggestion
     <div class="p-heading-icon--small">
        <div class="p-heading-icon__header">
          {{
            image(
            url="https://assets.ubuntu.com/v1/30037eac-datasheet-white.svg",
            alt="",
            width="32",
            height="28",
            hi_def=True,
            loading="lazy",
            attrs={"class": "p-heading-icon__img"},
            ) | safe
          }}
          <h3 class="p-heading--4">Datasheet</h3>
        </div>
        <p><a class="p-link--inverted" href="https://assets.ubuntu.com/v1/b5f9ae49-Enterprise_Kubernetes_Datasheet.pdf">Kubernetes for the enterprise</a></p>
      </div>
```

**Review by minkyngkm — APPROVED** [review]
LGTM @petesfrench! Nice work 🏅

File: templates/kubernetes/what-is-kubernetes.html (hunk 12 above)
Comment (petesfrench) [reply templates/kubernetes/what-is-kubernetes.html:207]: I don't think it will make a difference visually but can add it in anyway

**Pull request merged**